source/default/_patterns/01-atoms/svg/sprite-svg.css

```
@@ -0,0 +1,5 @@
+[class^='sprite-'] {
+  @apply w-6 h-6;
+
+  display: inline-block;
```

This could use the Tailwind class for inline block as well.

```suggestion
  @apply w-6 h-6 inline-block;
```

---

flerp